### PR TITLE
add more non-human genetic diseases

### PR DESCRIPTION
**Pull request by katiermullen**
addresses #7271

### CODE REVIEW

**Comment by katiermullen** [comment]
@sabrinatoro could you please review this PR for me at your convenience?

Thank you!

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -567972,6 +568238,434 @@ intersection_of: MONDO:0700097 MONDO:0009561 ! cross-species analog alpha-mannos
...
+
+[Term]
+id: MONDO:1010898
+name: sphingomyelinase deficiency, cattle
+def: "Neurological disease characterized by ataxia, hypermetria, wide-base stance, and strabismus that occurs in cattle due to a sphingomyelinase deficiency resulting in the accumulation of sphingomyelin in neurons and macrophages." [PMID:18424834]
+synonym: "Niemann-Pick disease, type A, cattle" RELATED [OMIA:001795-9913]
+xref: OMIA:001795-9913 {source="MONDO:equivalentTo"}
+is_a: MONDO:1010862 {source="OMIA:001795-9913", source="https://orcid.org/0000-0002-5002-8648"} ! lysosomal storage disease, cattle
+is_a: MONDO:1011321 {source="OMIA:001795-9913", source="https://orcid.org/0000-0002-5002-8648"} ! hereditary disease, non-human animal
+relationship: in_taxon NCBITaxon:9913 {source="OMIA:001795-9913"} ! Bos taurus
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:568592]: same as above with cattle

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -569409,6 +570140,54 @@ relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/
...
+[Term]
+id: MONDO:1011040
+name: oculocutaneous albinism, SLC45A2-related, cattle
+def: "Any oculocutaneous albinism that occurs in cattle due to a mutation in the SLC45A2 gene." [MONDO:patterns/nonhuman_disease_series_by_gene]
+synonym: "coat color albinism, oculocutaneous type IV, cattle" RELATED [OMIA:001821-9913]
+xref: OMIA:001821-9913 {source="MONDO:equivalentTo"}
+is_a: MONDO:1011321 {source="OMIA:001821-9913", source="https://orcid.org/0000-0002-5002-8648"} ! hereditary disease, non-human animal
+is_a: MONDO:1012579 {source="OMIA:001821-9913", source="https://orcid.org/0000-0002-5002-8648"} ! oculocutaneous albinism, non-human animal
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/ncbigene/538746 {source="OMIA:001821-9913", source="https://orcid.org/0000-0002-5002-8648"} ! SLC45A2
+relationship: in_taxon NCBITaxon:9913 {source="OMIA:001821-9913"} ! Bos taurus
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:570188]: same as above

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -571849,6 +572764,31 @@ relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/
...
+[Term]
+id: MONDO:1011257
+name: achromatopsia, CNGB3-related, cattle
+def: "Any achromatopsia that occurs in cattle due to a mutation in the CNGB3 gene." [MONDO:patterns/nonhuman_disease_series_by_gene]
+synonym: "achromatopsia-3 CNGB3-related, cattle" RELATED [OMIA:001365-9913]
+xref: OMIA:001365-9913 {source="MONDO:equivalentTo"}
+is_a: MONDO:1011321 {source="OMIA:001365-9913", source="https://orcid.org/0000-0002-5002-8648"} ! hereditary disease, non-human animal
+is_a: MONDO:1012578 {source="OMIA:001365-9913", source="https://orcid.org/0000-0002-5002-8648"} ! achromatopsia, non-human animal
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/ncbigene/519225 {source="OMIA:001365-9913", source="https://orcid.org/0000-0002-5002-8648"} ! CNGB3
+relationship: in_taxon NCBITaxon:9913 {source="OMIA:001365-9913"} ! Bos taurus
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:572789]: same as above

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -571947,6 +572984,44 @@ relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/
...
+id: MONDO:1011276
+name: retinitis pigmentosa, RP1-related, cattle
+def: "Any retinitis pigmentosa that occurs in cattle due to a mutation in RP1 gene." [MONDO:patterns/nonhuman_disease_series_by_gene]
+synonym: "retinitis pigmentosa 1, cattle" RELATED [OMIA:002029-9913]
+xref: OMIA:002029-9913 {source="MONDO:equivalentTo"}
+is_a: MONDO:1011259 {source="OMIA:002029-9913", source="https://orcid.org/0000-0002-5002-8648"} ! retinitis pigmentosa, cattle
+is_a: MONDO:1011321 {source="OMIA:002029-9913", source="https://orcid.org/0000-0002-5002-8648"} ! hereditary disease, non-human animal
+relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/general/qc-single-child.sparql
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/ncbigene/280916 {source="OMIA:002029-9913", source="https://orcid.org/0000-0002-5002-8648"} ! RP1
+relationship: in_taxon NCBITaxon:9913 {source="OMIA:002029-9913"} ! Bos taurus
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:573010]: same as above

**Review by sabrinatoro — APPROVED** [review]
@katiermullen I approve this PR. Please feel free to merge. 
I added some comments about using "Bos taurus" when referring to "cattle". Based on VBO (haha), we know that cattle include more than Bos taurus. However, the NCBIgene is from Bos taurus (and not any "Bos"). Therefore I think it is ok to leave as is. IF there is a case where the gene and the disease are for a different Bos, we can review this decision, but I think it is ok.

**Comment by katiermullen** [comment]
Thank you @sabrinatoro. Your point is noted and I will document our decision to keep the taxon as Bos taurus as long as the NCBIGene points to Bos taurus, although it is possible that the condition could be reported in a different species of Bos. If the NCBIGene points to a different Bos, I will report that species for in_taxon. Thank you for catching this!

**Pull request merged**